src/keymap.cpp

```diff
@@ -139,7 +139,7 @@ static op_desc opdescs[] = {
 	},
 	{
 		OP_SETTAG, "set-tag", "t",
-		_("Select tag"),
+		_("Set tag"),
```

Comment:
Wouldn't that also reintroduce the typo in `src/keymap:37` and the translation files? I'm pretty new to git in general, and should probably have split my commits further... I think I have it sorted out and will make a new commit now.